.github/workflows/dcr-container.yml

```diff
@@ -0,0 +1,52 @@
+name: DCR Build Container Image
+
+on:
+  push:
+    paths-ignore:
+      - "apps-rendering/**"
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      packages: write
+
+    outputs:
+      container-image: ${{ steps.push.outputs.registry-path }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@3.5.1
+
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+
+      - name: Generate production build
+        run: make build
+        working-directory: dotcom-rendering
+
+      - name: Build Container Image
+        id: build_image
+        uses: redhat-actions/buildah-build@v2.11
+        with:
+          image: dotcom-rendering
+          tags: ${{ github.sha }} ${{ env.GITHUB_REF_SLUG }}
+          context: ./
+          containerfiles: ./dotcom-rendering/Containerfile
```

Comment:
Wonder if we could even rename this to `Containerfile.ci` or something that makes it clear it's not for production?